### ISSUE TITLE
Create table of linked weapon groups and ghost heat penalties.

### ISSUE THREAD

**Issue by sestrand** [lisongmechlab/lsml#791]
**LSML version:** 2.0.15
**Screen resolution:** 3440x1440
**Java version:** 8uX

new features based on the demise of smurfy.  From an internet archive... this was always useful to me:

![Screenshot 2022-08-31 125312](https://user-images.githubusercontent.com/46074852/187746427-458499ce-712e-46bf-88b3-cfa105ebca4c.png)

There are also a few columns in weapons that LSML does not have, but might be helpful to pull out for a weapon browser. such as ammo/t and projectile velocity.  might be good to display ammo in terms of seconds worth also.  

![Screenshot 2022-08-31 125958](https://user-images.githubusercontent.com/46074852/187747551-ed8e5552-d0a7-4bb8-ac55-8d095f0b6244.png)
![Screenshot 2022-08-31 130035](https://user-images.githubusercontent.com/46074852/187747666-fdb1827d-b6f0-4477-8177-601b94b6c7f8.png)

smurfy also used to break out the ammo in its own table:
![Screenshot 2022-08-31 130314](https://user-images.githubusercontent.com/46074852/187748254-d9ab0746-13fc-46f8-b238-5a2f88bebf2c.png)

for all of these, there are parts of LSML that could probably just display the values (in tools tips for ammo or engines or whatever, perhaps). but I did like having a way to see the bonus heat and how quickly it gets bad for any certain weapon.

other features smurfy had may not be in the scope of LSML such as maps.

**Comment by LiSongMWO** [comment]
Most of this info is already in the LSML data files with the exception of cost and explosion damage.

I believe that the cost was manual data entry by Smurfy, and I'm categorically opposed to that as that's too much work for me. If there's a way of finding this from the data files then that's A-OK to add to tooltips or even compute a total cost for a loadout etc.

Explosion damage I don't know how it's calculated, I think there's a formula somewhere?

The weapons table is pretty much covered already. Projectile speed is shown under the "Spd" header in LSML. Open to suggestions for a better name that doesn't make the colmun too wide.

Ammo/t can be added but I never found that useful as I use the weapons summary to look at my total damage potentinal and combat time instead of counting tons. I feel like there's probably some better metric that would make the whatever decision the user is trying to make easier. Like, "How many tons (weapon+ammo) for 200 damage potential." for example.

**Comment by sestrand** [comment]
I agree most is already there, and it is certainly all in the LSML database ... LSML does not display some of this as obviously as a Smurfy but does a better job with other things.  The threads regarding the loss of smurfy did have some shout outs to LSML.  But the notable table that people were missing was the heat map.

I am in complete agreement than manually entering data to support LSML is a non-starter.  I was just showing the historical tables to provide some ideas.

I vaguely recall the component / ammo explosion damage listed somewhere.  might be in a PGI file or might have just been in a release note.

**Comment by LiSongMWO** [comment]
Ok I can see the ghost heat matrix being useful and possible to add as a new page on the main window. In order to make sure that all issues have a clear condition for when they can be closed I'll define this issue as implementing that table.

Could you file any additions that are necessary to the existing tables as separate FRs? It's more likely they'll get implemented that way when I'm looking for something small I can knock out for the next release I'm working on without committing to a large project.
